shell/platform/android/io/flutter/embedding/engine/systemchannels/MouseCursorChannel.java

```diff
@@ -0,0 +1,86 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine.systemchannels;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.json.JSONArray;
+
+import java.util.HashMap;
+
+import io.flutter.Log;
+import io.flutter.embedding.engine.dart.DartExecutor;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.StandardMethodCodec;
+
+/**
+ * System channel that receives requests for mouse cursor behavior, e.g., set as system cursors.
+ */
+public class MouseCursorChannel {
+  private static final String TAG = "MouseCursorChannel";
+  public static final String CHANNEL_NAME = "flutter/mousecursor";
```

Comment:
This is fine, but for consistency, we seem to just use the string literal directly in the MethodChannel constructor for other channels.